manifests/dns.pp

```diff
@@ -27,5 +27,5 @@
   Array[Simplib::Host] $servers = []
 ){
   assert_private()
-  validate_net_list($servers)
+  simplib::validate_net_list($servers)
```

Comment:
Why do we use validate_net_list instead of just declaring the variables a Simplib::Netlist type.  Wouldn't that accomplish the same thing?

Reply:
I'll remove those calls as they don't provide much value

Reply:
Actually, the simplib::validate_net_list catches errors in which a user fat-fingers an IPV4 address.  For example, 1.2.3.600.  So, I think I would like to keep it.